bot/cogs/watchchannels/talentpool.py

```diff
@@ -0,0 +1,276 @@
+        """
+        Relay messages sent by the given `user` to the `#big-brother-logs` channel.
+
+        A `reason` for adding the user to BigBrother is required and will displayed
+        in the header when relaying messages of this user to the watchchannel.
+        """
+        if user.bot:
+            e = Embed(
+                description=f":x: **I'm sorry {ctx.author}, I'm afraid I can't do that. I only watch humans.**",
+                color=Color.red()
+            )
+            await ctx.send(embed=e)
+            return
+
+        if isinstance(user, Member) and any(role.id in STAFF_ROLES for role in user.roles):
+            e = Embed(
+                description=f":x: **Nominating staff members, eh? You cheeky bastard.**",
+                color=Color.red()
+            )
+            await ctx.send(embed=e)
+            return
+
+        if not await self.fetch_user_cache():
+            log.error(f"Failed to update user cache; can't watch user {user}")
+            e = Embed(
+                description=f":x: **Failed to update the user cache; can't add {user}**",
+                color=Color.red()
+            )
+            await ctx.send(embed=e)
+            return
+
+        if user.id in self.watched_users:
+            e = Embed(
+                description=":x: **The specified user is already in the TalentPool**",
+                color=Color.red()
+            )
+            await ctx.send(embed=e)
+            return
+
+        # Manual request with `raise_for_status` as False because we want the actual response
+        session = self.bot.api_client.session
+        url = self.bot.api_client._url_for(self.api_endpoint)
+        kwargs = {
+            'json': {
+                'actor': ctx.author.id,
+                'reason': reason,
+                'user': user.id
+            },
+            'raise_for_status': False,
+        }
+        async with session.post(url, **kwargs) as resp:
+            response_data = await resp.json()
+
+            if resp.status == 400 and response_data.get('user', False):
+                e = Embed(
+                    description=":x: **The specified user can't be found in the database tables**",
+                    color=Color.red()
+                )
+                await ctx.send(embed=e)
+                return
+
+            resp.raise_for_status()
+
+        self.watched_users[user.id] = response_data
+        e = Embed(
+            description=f":white_check_mark: **Messages sent by {user} will now be relayed to TalentPool**",
+            color=Color.green()
+        )
+        await ctx.send(embed=e)
+
+    @nomination_group.command(name='history', aliases=('info', 'search'))
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def history_command(self, ctx: Context, user: Union[User, proxy_user]) -> None:
+        """Shows the specified user's nomination history"""
```

Comment:
I've noticed some docstrings have full stops and some don't. What's it gonna be?

Reply:
Full stops. I don't think we enforce the rule under the new linting rules, because we want to allow other punctuation marks as well, but we still concluded that we prefer to have some kind of punctuation mark at the end of the summary line, since it will also be used in the help embeds.

Reply:
Sounds good. Not sure if you've made the changes yet so I'll let you resolve this when it's ready.

Reply:
Looks like you did it actually so I'll resolve.